### DISCUSSION TITLE
Adding Descriptions to Controls with Hover Modals in GrapesJS

### DISCUSSION THREAD

**Discussion by asimex** [GrapesJS/grapesjs#5975] [Q&A]
I am looking for a way to provide descriptive information about various controls in the GrapesJS editor. Specifically, I would like to implement a feature where users can hover over certain controls, such as the plus button below the classes on the right side, and see a modal popup that explains what the control does.

For example, when a user hovers over the plus sign, a modal should appear providing a description of its functionality. I would like to extend this feature to other controls as well.

Is there a recommended approach or built-in functionality in GrapesJS that would allow me to achieve this? Any guidance or examples on how to implement this would be greatly appreciated.

Thank you for your help!

**Comment by artf** [comment]
You should probably go with your custom UI here
